### ISSUE TITLE
Autocomplete on forms

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/capstone#503]
Investigate why the login/registration forms are failing Axe accessibility tests with:
![image](https://user-images.githubusercontent.com/11020492/45898221-50590700-bda7-11e8-9617-9d7180100f96.png)

https://dequeuniversity.com/rules/axe/3.1/autocomplete-valid?application=AxeChrome

**Comment by jcushman** [comment]
Looks like we don't have autocomplete on password fields anymore
